src/construction.h

```diff
@@ -53,6 +53,7 @@ struct construction {
         int print_time( const catacurses::window &w, int ypos, int xpos, int width, nc_color col ) const;
         std::vector<std::string> get_folded_time_string( int width ) const;
         float time_scale() const; //result of construction scaling option
+        const std::string desc() const;
```

Comment:
Btw. why is this returning a *const* object?

Reply:
Done like in #26038 to handle `warning: returning reference to temporary`.
`std::string const & --> std::string const`
I should remove const in both cases, right?

Reply:
Okay. Thank you.